### DISCUSSION TITLE
How to use OwnPlate?

### DISCUSSION THREAD

**Discussion by nordicstar** [Nakajima-Foundation/ownplate#1204] [General]
Hi, I made it this far - is Own Plate in English, one question, second, I figured out enough to get here, but realistically for someone, me, 70 year old woman, with not much coding experience, how can I install Own Plate to use, if it is in English - can someone give me this information?  Thank You -

**Comment by ys64** [comment]
Hello @nordicstar 

Thank you for reaching out! I'm glad to hear you've made progress with OwnPlate.

To answer your question, yes, OwnPlate is available in English. Regarding the installation process, I understand that it can be a bit challenging, especially if you're not very familiar with coding. I would be happy to assist you with the installation steps.

Could you let me know if you're still working on this? If you need help, I can provide detailed instructions or walk you through the process.

Looking forward to hearing from you!

Best regards
